junit5-support/src/main/kotlin/io/specmatic/test/reports/ReportProcessor.kt

```diff
@@ -2,10 +2,11 @@ package io.specmatic.test.reports
 
 import io.specmatic.core.ReportConfiguration
 import io.specmatic.core.SpecmaticConfig
+import io.specmatic.test.ContractTestSettings
 import io.specmatic.test.reports.renderers.ReportRenderer
 
 interface ReportProcessor<T> {
-    fun process(specmaticConfig: SpecmaticConfig)
+    fun process(specmaticConfig: SpecmaticConfig, settings: ContractTestSettings)
```

Comment:
Can you accept settings in the constructor of this object instead of as a parameter.